vms/evm/contract_message.go

```diff
@@ -23,13 +26,24 @@ func NewContractMessage(logger logging.Logger, subnetInfo config.SourceSubnet) *
 }
 
 func (m *contractMessage) UnpackWarpMessage(unsignedMsgBytes []byte) (*vmtypes.WarpMessageInfo, error) {
+	// This function may be called with raw UnsignedMessage bytes or with ABI encoded bytes as emitted by the Warp precompile
```

Comment:
Nit: Could we include a comment above the function definition here, documenting how it can be used? We can leave the comments explaining the reasoning for details of how it's implemented within the body.